drivers/usb/device/Kconfig

```diff
@@ -39,6 +39,39 @@ config USB_DC_STM32
 	help
 	  STM32 family USB device controller Driver.
 
+config USB_DC_STM32_IRQ_PRI
+	int "USB STM32 Driver Interrupt priority"
+	depends on USB_DC_STM32
+	default 3
+	help
+	  USB interrupt priority.
+
+config USB_DC_STM32_EP_NUM
```

Comment:
Please remove this.

Reply:
My mistake. I'll fix this and update it.